source/guides/references/protractor-migration-guide.md

```diff
@@ -0,0 +1,565 @@
+
+This feature is available in Protractor and requires you to configure your application with multiple options (i.e., `sharedTestFiles`, `maxInstances`, etc.). An example is provided below:
+
+**Before: Protractor**
+
+```js
+// Example from https://developers.perfectomobile.com/display/PD/Protractor+parallel+execution
+// An example configuration file.
+exports.config = {
+
+  seleniumAddress: 'https://cloudName.perfectomobile.com/nexperience/perfectomobile/wd/hub',
+
+  // Capabilities to be passed to the webdriver instance.
+  capabilities: {
+    browserName: 'chrome',
+
+    // allows different specs to run in parallel.
+    // If this is set to be true, specs will be sharded by file
+    // (i.e. all files to be run by this set of capabilities will run in parallel).
+    // Default is false.
+    sharedTestFiles: true,
+
+    // Maximum number of browser instances that can run in parallel for this
+    // set of capabilities. This is only needed if shardTestFiles is true.
+    // Default is 1.
+    maxInstances: 2,
+
+    // Cloud capabilities
+    user: 'user@perfectomobile.com',
+    password: 'password',
+
+    // Device capabilities
+    platformName: 'Android',
+    manufacturer: 'Samsung',
+    model: 'Galaxy S5'
+  },
+
+  // Framework to use. Jasmine is recommended.
+  framework: 'jasmine',
+
+  // Spec patterns are relative to the current working directly when
+  // protractor is called.
+  specs: ['spec.js', 'spec2.js'],
+
+  // Options to be passed to Jasmine.
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000
+  }
+}
+```
+
+However, with Cypress, all you need to do is pass the `--parallel` and `--record` flag to `cypress run`, and it will take care of the rest for you:
+
+**After: Cypress**
+
+```bash
+$ cypress run --record --parallel
+```
+
+For more information, check out our [official docs on parallelization](https://docs.cypress.io/guides/guides/parallelization.html#Overview)!
+
+### Element Explorer
+
+For those who are big fans of [Protractor's Element Explorer functionality](https://www.protractortest.org/#/debugging#enabled-control-flow), Cypress also provides you with a Selector Playground that allows you to:
+
+- Determine a unique selector for an element
+- See what elements match a given selector
+- See what element matches a string of text
+
+For more information, check out our official docs on the [Selector Playground](https://docs.cypress.io/guides/core-concepts/test-runner.html#Selector-Playground).
+
+## Next Steps
+
+For more information on how to create end-to-end tests with Cypress, be sure to check out [our official documentation here](https://docs.cypress.io/guides/overview/why-cypress.html).
+
+{% note warning %}
+If you see any inaccuracies with this or feel something has been misrepresented, please [submit an issue here](https://github.com/cypress-io/cypress-documentation/issues/new).
+{% endnote %}
```

Comment:
We typically have a See Also section at the bottom of guides - if you think there are some things to link to from here that may be helpful when they make it to the end. See other docs.